docs/index.rst

```diff
@@ -1,46 +1,34 @@
-.. PyFFI documentation master file, created by sphinx-quickstart on Fri Oct 24 19:47:53 2008.
+.. PyFFI documentation master file, created by
+   sphinx-quickstart on Sat Jun 10 21:06:06 2017.
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to PyFFI!
```

Comment:
I would leave it the previous way. Also think we should have the blurb here.

Reply:
Not touched in the new branch